lib/ansible/modules/cloud/azure/azure_rm_storageaccount_keys.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2016 Amanvir Mundra, <@amanvirmundra>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: azure_rm_storageaccount_keys
+
+version_added: "2.6"
+
+short_description: Retrieve Azure Storage Account Keys.
+
+description:
+    - Retrive keys for a Storage Account.
+
+options:
+    resource_group:
+        description:
+            - This is the name of the resource group for which you want to retrieve Keys
+        required: true
+        default: null
+    name:
+        description:
+            - Name of the Storage Account instance.
+        required: true
+        default: null
+        aliases:
+            - resource_group_name
+
+extends_documentation_fragment:
+    - azure
+
+author:
+    - Amanvir Mundra (@amanvirmundra)
+
+'''
+
+EXAMPLES = '''
+- name: Get Storage account keys
+  azure_rm_storageaccount_keys:
+    resource_group: resource_group
+    name: name
+'''
+
+RETURN = '''
+key1:
+    description: Key1 for azure storage account
+    returned: always
+    type: str
+key2:
+    description: Key2 for azure storage account
+    returned: always
+    type: str
+'''
+
+from ansible.module_utils.azure_rm_common import AzureRMModuleBase
+try:
+    from msrestazure.azure_exceptions import CloudError
+except ImportError:
+    # This is handled in azure_rm_common
+    pass
+
+
+class AzureRMStorageAccountKeys(AzureRMModuleBase):
+    def __init__(self):
+
+        self.module_arg_spec = dict(
+            resource_group=dict(type='str', aliases=['resource_group_name'], required=True),
+            name=dict(type='str', required=True)
+        )
+        self.results = dict(
+            changed=False,
+            key1=None,
```

Comment:
"key1" and "key2" here is in lowercase, while later there's results['Key1'] and results['Key2']

Reply:
Got it. This should go away once I do the looping change. Thanks.